how-to-contribute/contributor-roadmap-participating.md

```diff
@@ -1,18 +1,18 @@
 # Contributor Roadmap - Participating
 
-CHAOSS is not comprised of only software, so it's important that you have a feel for all the moving parts of our project. If you've gone through the steps on our [Newcomer Quickstart](https://chaoss.community/kb-getting-started/), you're ready to start participating a little more! That's awesome! 
+There are several ways you can contribute to CHAOSS in different areas. 
```

Comment:
```suggestion
There are several ways you can contribute to CHAOSS. 
```

'in different areas' seemed the same as 'several ways', so I think this can be simplified?